studio/components/layouts/ProjectLayout/UpgradingState/UpgradingState.tsx

```diff
@@ -155,19 +161,19 @@ const UpgradingState = () => {
                         return (
                           <div key={message.key} className="flex items-center space-x-4">
                             {isCurrent ? (
-                              <div className="h-5 w-5 flex items-center justify-center rounded-full">
+                              <div className="flex items-center justify-center w-5 h-5 rounded-full">
                                 <IconLoader
                                   size={20}
                                   className="animate-spin text-scale-1100"
                                   strokeWidth={2}
                                 />
                               </div>
                             ) : isCompleted ? (
-                              <div className="h-5 w-5 flex items-center justify-center rounded-full border bg-brand-800 border-brand-700">
```

Comment:
Are these reorderings from prettier "format on save" or something? Or just your initiative?
Should we enable the _format on save_ as default for everyone so tw class order is always consistent?

Reply:
Uhh I have it disabled for Supabase usually, but because I was doing self-hosted development it re-enabled itself 😬

Please ignore!

(we might look at enabling it for everyone in future, but in studio we're writing styles in many different ways – `clsx()` for example – which would break the formatting anyway)